src/Plugins/AeroAtomIspCamera/CMakeLists.txt

```diff
@@ -1,7 +1,9 @@
-file(GLOB SRC_FILES *.cpp)
+file(GLOB SRC_FILES
+    CameraDeviceAeroAtomIsp.cpp
+    PluginAeroAtomIsp.cpp)
```

Comment:
I think you don't need `file(GLOB SRC_FILES ..)` at all anymore because you can just list the files in `add_library()` directly. Or is this on purpose?

Reply:
Agree that it is not necessary. That's how it was implemented in project that was used as a template.